frontend/components/feedback-task/header/metadata-filter/MetadataFilter.vue

```diff
@@ -93,11 +99,38 @@ export default {
         this.metadataFilters.convertToRouteParam()
       );
     },
+    openCategoryFilter(category, e) {
+      e.stopPropagation();
+      this.visibleDropdown = this.visibleDropdown ? false : true;
+      this.selectMetadataCategory(category);
+    },
+    removeCategoryFilters(category, e) {
+      e.stopPropagation();
+      const clickedCategory = this.metadataFilters.metadata.find(
```

Comment:
Instead to add business logic in this component try with:
`this.metadataFilters.findByCategory(category)` 
we have this business logic into our domain